nservicebus/upgrades/release-policy.md

```diff
@@ -79,3 +80,45 @@ Since all critical issues will be back-ported, choose to upgrade when convenient
 ### Major
 
 Since a new `major` version will contain breaking changes and will likely require modifications to the consuming code. Because of the extent of code changes in a major version, it is recommendation that a full regression test of the system is done.
+
+
+### Release quality
+
+The release cycle consists of the following quality stages:
+
+ * Alpha
+ * Beta (Optional)
+ * Release Candidate (Optional)
+ * Stable
+
+
+### Alpha
+
+ * Binaries are considered unstable.
+ * APIs might change without notice.
+ * Packages are used for internal development, updating docs, etc.
+
+
+### Beta
+
+ * The component is feature complete.
+ * Likely to contain a number of known or unknown bugs.
+ * Performance and stability testing has not been fully completed.
+ * **No** "go live" license is provided and production use is **not supported**.
+ * All public APIs should be stable but may change based on feedback from consumers.
+
+
+#### Open and closed beta
```

Comment:
Why does this distinction matter?
People who have access to this are public. If you're adding this, how about you also add something that states that, if someone is interested in being part of the closed group beta, they should contact us?